markdownlint.json

```diff
@@ -1,24 +1,29 @@
 {
   "ignorePatterns": [
     {
-      "pattern": "^https:\/\/github.com\/cognitedata\/.+\/issues\/.+"
+      "pattern": "^https://github.com/cognitedata/.+/issues/.+"
     },
     {
-      "pattern": "^https:\/\/github.com\/cognitedata\/.+\/compare\/.+"
+      "pattern": "^https://github.com/cognitedata/.+/compare/.+"
     },
     {
-      "pattern": "^https:\/\/github.com\/cognitedata\/.+\/commit\/.+"
+      "pattern": "^https://github.com/cognitedata/.+/commit/.+"
     },
     {
-      "pattern": "^http:\/\/localhost.+"
+      "pattern": "^http://localhost.+"
     },
     {
-      "pattern": "^https:\/\/localhost.+"
+      "pattern": "^https://localhost.+"
     },
     {
       "pattern": "^mailto:.+"
+    },
+    {
+      "pattern": "^https://doc.cognitedata.com+"
```

Comment:
https://cognitedata.slack.com/archives/CN9QYQRSR/p1695039928231879
This is a bit sad. The URL initially returns a 404 now, before loading the correct page. I can't think of a reasonable fix for this right now, aside from ignoring these :(

Reply:
nevermind!